App.js

```diff
@@ -31,30 +31,38 @@ import LoginPage from './components/auth/loginPage';
 import SignupPage from './components/auth/signupPage';
 
 import fakeUser from './data/profileData.js';
-import data from './data/data';
 const Tab = createBottomTabNavigator();
 
 export default function App() {
   // Determine user has light/dark theme on phone
   let phoneTheme = true;
   const colorScheme = Appearance.getColorScheme();
-  if (colorScheme === 'dark') {
+  if (colorScheme === 'light') {
     phoneTheme = false;
 }
 
   // Add State that will be shared globally here
-  const [name, setName] = useState(fakeUser.first_name);
+  const [userEmail, setUserEmail] = useState('');
+  const [userId, setUserId] = useState('');
+  const [fetchdata, setFetchData] = useState('');
+  const [friendsList, setFriendsList] = useState('');
+  const [token, setToken] = useState('');
+
   const [profileSettingsOpen, setProfileSettingsOpen] = useToggle(false);
   const [editProfile, setEditProfile] = useToggle(false);
   const [logoutModalOpen, setLogoutModalOpen] = useToggle(false);
   const [changePassModalOpen, setChangePassModalOpen] = useToggle(false);
   const [isDarkTheme, setIsDarkTheme] = useToggle(phoneTheme);
-  const [isLoggedIn, setLoggedIn] = useToggle(false);
+  const [isLoggedIn, setLoggedIn] = useState(false);
+
   const [email] = useState(fakeUser.email);
   const [currentUser, setCurrentUser] = useState(5);
-  const [userData, setUserData] = useState(data);
+  const [realUserData, setRealUserData] = useState({});
+
   const [authPage, setAuthPage] = useState('login');
 
+  const [checkout,setCheckout] = useState({});
```

Comment:
I cannot tell by the name what this does and I do not see it uses anywhere. Consider adding a description to this and any variables/functions that are not self-descriptive.